src/gluonts/evaluation/_base.py

```diff
@@ -554,4 +625,21 @@ def __call__(
             all_agg_metrics, all_metrics_per_ts
         )
 
+        if len(self.target_agg_funcs) != 0:
+            multivariate_metrics = {
+                key_value[0]: self.calculate_aggregate_multivariate_metrics(
+                    ts_iterator_set[-(index + 1)],
+                    fcst_iterator_set[-(index + 1)],
+                    key_value[1],
+                )
+                for index, key_value in enumerate(
```

Comment:
we can write `key_value` as a pair like this:
```
for index, (key, value) in enumerate(
```

Reply:
Also maybe rename `(key, value)` as (`agg_fun_name`, `agg_fun`).

Reply:
Thanks! Good suggestion!